ctx.go

```diff
@@ -456,8 +456,15 @@ func (c *Ctx) Cookie(cookie *Cookie) {
 	// i.e. cookie supposed to last beyond browser session
 	// refer: https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_the_lifetime_of_a_cookie
 	if !cookie.SessionOnly {
-		fcookie.SetMaxAge(cookie.MaxAge)
-		fcookie.SetExpire(cookie.Expires)
+		// Cookies are session cookies if they do not specify the Expires or Max-Age attribute.
+		// refer: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie
+		// phrase from MDN: "If both Expires and Max-Age are set, Max-Age has precedence."
+		// Skipping setting 0 time and seconds to MaxAge and Expires
+		if cookie.MaxAge != 0 {
```

Comment:
now the question is how to make sure that you send an expire with 0 when defining the cookie 

shouldn't it rather be adjusted within the session middleware, that as soon as you send 0, the sessionOnly propertie is used when setting the cookie, instead of adjusting the general feature to set the cookie ?